jani_generator/test/test_systemtest_scxml_to_jani.py

```diff
@@ -215,7 +215,7 @@ def _test_with_main(self, main_xml: str, folder: str, property_name: str, succes
         pos_res = "Result: 1" if success else "Result: 0"
         neg_res = "Result: 0" if success else "Result: 1"
         run_smc_storm_with_output(
-            f"--model {ouput_path} --property-name {property_name}",
+            f"--model {ouput_path} --properties-names {property_name}",
```

Comment:
Thats why you need the new smc-storm version, right?

Reply:
Double plural ?!

Reply:
I wasn't sure, either... But the internet says it should be fine :)